runtime/vm/JFRConstantPoolTypes.hpp

```diff
@@ -1246,6 +1274,64 @@ class VM_JFRConstantPoolTypes {
 		gcConfiguration->heapAddressBits = J9JAVAVM_REFERENCE_SIZE(vm) * 8;
 	}
 
+	static uintptr_t processNativeLibrariesCallback(const char *libraryName, void *lowAddress, void *highAddress, void *userData)
+	{
+		if (NULL == strstr(libraryName, ".so")) {
+			return 0;
+		}
+		VM_JFRConstantPoolTypes *constantPoolTypes = static_cast<VM_JFRConstantPoolTypes *>(userData);
+		J9Pool *nativeLibrariesTable = constantPoolTypes->_nativeLibrariesTable;
+		NativeLibraryEntry *firstNativeLibraryEntry = constantPoolTypes->_firstNativeLibraryEntry;
+		NativeLibraryEntry *previousNativeLibraryEntry = constantPoolTypes->_previousNativeLibraryEntry;
+		NativeLibraryEntry *entry = firstNativeLibraryEntry;
+		PORT_ACCESS_FROM_JAVAVM(constantPoolTypes->_vm);
+		while (NULL != entry)
```

Comment:
formatting `while (cond) {`

Reply:
see https://github.com/eclipse-omr/omr/blob/master/doc/CodingStandard.md for coding guidelines